packages/manager/modules/billing/src/payment/fidelity/billing-fidelity.controller.js

```diff
@@ -1,142 +1,161 @@
-export default /* @ngInject */ (
-  $scope,
-  $filter,
-  $timeout,
-  $translate,
-  BillingFidelity,
-  BillingmessageParser,
-  BillingdateRangeSelection,
-  coreConfig,
-) => {
-  $scope.fidelityLoading = false;
-  $scope.fidelityAccountLoading = false;
-  $scope.fidelityAccount = null;
-  $scope.currency = null;
-
-  function getItems() {
-    $scope.fidelityLoading = true;
-
-    BillingFidelity.getMovements(
-      BillingdateRangeSelection.dateFrom,
-      BillingdateRangeSelection.dateTo,
-    )
-      .then((movements) => {
-        $scope.tasksId = movements.reverse();
-      })
-      .catch((data) => {
-        $scope.setMessage(
-          $translate.instant('fidelity_get_movements_error'),
-          data.data,
-        );
-        $scope.loaders.tasks = false;
-      })
-      .finally(() => {
-        $scope.fidelityLoading = false;
+export default class {
```

Comment:
**suggestion**: class can be named.
```suggestion
export default class BillingFidelityController {
```

Reply:
We will add it via a separate Pull Request.